docs/modules/ROOT/pages/custom-attributes.adoc

```diff
@@ -0,0 +1,99 @@
+= Option: Custom attributes string
+:navtitle: Custom attributes string
+
+The extension option *Custom attributes string* is a space-separated list of attribute settings.
+
+You may use the above string, and other methods to set attributes that affect the document preview.
+For all methods and when they are applied, visit the xref:features.adoc#set-attributes[Set attributes] section on the Features page.
```

Comment:
There's no "set-attributes" id in the features page.

Reply:
fixed to use set-asciidoc-attributes